src/js/components/Page/Page.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useMemo } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Box } from '../Box';
+import { PageContext } from './PageContext';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+import { PagePropTypes } from './propTypes';
+
+const Page = ({ kind, ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  const value = useMemo(
+    () => ({
```

Comment:
What about structuring the theme to be `theme.page[kind][size].pad`? I think this would align a bit better with how we structure the theme namespace with state before property, so we can put all properties appropriate for a state together, as opposed to put all states for a property together.

This would also allow passing `theme.page[kind][size]` as the value for the PageContext, which would allow adding other properties to the theme and have them get passed along.

Reply:
Yes, changed in latest commit